lib/rules/named-grid-areas-no-invalid/index.js

```diff
@@ -45,7 +44,13 @@ function rule(actual) {
 					return;
 				}
 
-				areas.push(_.compact(tokenValue.trim().split(' ')));
+				areas.push(
+					tokenValue
+						.trim()
+						.split(' ')
+						// keep only truthy values
+						.filter((t) => t),
```

Comment:
[suggestion] `split()` returns an array of strings, so how about clarifying the use of string argument as below?

```suggestion
						.filter((s) => s.length > 0),
```

Reply:
Nice, I like that better.